### ISSUE TITLE
Cannot render accessibility indicators with very few distinct values

### ISSUE THREAD

**Issue by abyrd** [conveyal/analysis-ui#628]
This occurs especially when comparing two sets of results with almost no differences. If we ask for 5 colors and there are less than 5 distinct accessibility (accessibility difference) values it fails:
```
Error: cannot generate more classes than there are data values
    at ckmeans (ckmeans.js:220)
    at ckmeans.js:39
    at next (regional.js:95)
    at fetch.js:285
    at fetch.js:205
    at <anonymous>
```

**Comment by abyrd** [comment]
Some insights: 

The ckmeans clustering is performed by the SimpleStatistics library. It would be nice if it would just produce less categories when it didn't have enough distinct values to cluster, be we can't directly change its behavior. 

According to https://simplestatistics.org/docs/#ckmeans it will throw an error in this situation. We need to catch the error and provide alternate behavior.

```
  try {
      const clusters = ssCkmeans(filtered, nBreaks)      
  } catch (error) {
      // Fallback: return one break per value
  }
```

We need to make sure this is the only situation in which an error is thrown, or somehow check the error type. We then need to find all the distinct values in the original or filtered data and make one category per value, or one category for each of the first N values and one catch-all for all the rest of the values.

We can see where these breaks are used in ChoroplethColorizer, it should accept variable length breaks but expects the array of colors to be the same length.

**Comment by ansoncfit** [comment]
Closed by https://github.com/conveyal/analysis-ui/pull/1247
